### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by Chenxy517**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,329 +1,34 @@
...
+@ARTICLE{gensync,
+  author={Boškov, Novak and Trachtenberg, Ari and Starobinski, David},
   journal={IEEE Transactions on Network and Service Management},
-  volume={19},
-  number={4},
-  pages={4408--4423},
+  title={GenSync: A New Framework for Benchmarking and Optimizing Reconciliation of Data},
   year={2022},
-  publisher={IEEE}
+  volume={},
```
Comment (trachten) [inline README.md]: You should use the correct citation that has been deleted instead.

File: README.md [hunk 2]
```diff
@@ -1,329 +1,34 @@
...
-    * *InteractiveCPISync*
-* **setExpNumElems:** The maximum number of differences that you expect to be placed into your IBLT. If you are doing IBLTSetOfSets this is the number of child sets you expect
-    * *IBLTSync, OneWayIBLTSync & IBLTSetOfSets*
-* **setExpNumElemChild:** Set the upper bound for number of elements in each child set
-    * *IBLTSetOfSets*
-* **setDataFile:** Set the data file containing the data you would like to populate your GenSync with
-    * *Any sync you'd like to do this with*
-
-<a name="SyncTypes"></a>
-### Sync Types:
```
Comment (trachten) [inline README.md:281]: I think that these can stay.

**Review by trachten — COMMENTED** [review]
Looks good ... just minor edits to the README.

**Comment by Chenxy517** [comment]
Issues have been fixed

**Review by trachten — APPROVED** [review]

**Pull request merged**